snakes_cafe.py

```diff
@@ -0,0 +1,69 @@
+
+welcome_msg = """
+**************************************
+**    Welcome to the Snakes Cafe!   **
+**    Please see our menu below.    **
+**
+** To quit at any time, type "quit" **
+**************************************
+
+Appetizers
+----------
+Wings
+Cookies
+Spring Rolls
+
+Entrees
+-------
+Salmon
+Steak
+Meat Tornado
+A Literal Garden
+
+Desserts
+--------
+Ice Cream
+Cake
+Pie
+
+Drinks
+------
+Coffee
+Tea
+Unicorn Tears
+"""
+print(welcome_msg)
+
+cust_quest = """
+***********************************
+** What would you like to order? **
+***********************************
+"""
+
+def order():
+    print('order running')
+    print(cust_quest)
+    currentOrder = True
+    custOrder = {}
+    while currentOrder:
+        answer = input("> ")
+        if answer == "quit":
+            currentOrder = False
+    else:
```

Comment:
This else statement seems a bit out of place. If you're still fighting the bug here, I suggest doing a sweep of your indentations.

Since Python doesn't use bracketed {} codeblocks, lining up indentation correctly is important to ensure correct flow of the code.